apps/ocsp.c

```diff
@@ -160,7 +160,7 @@ const OPTIONS ocsp_options[] = {
     OPT_SECTION("Client"),
     {"url", OPT_URL, 's', "Responder URL"},
     {"host", OPT_HOST, 's', "TCP/IP hostname:port to connect to"},
-    {"port", OPT_PORT, 'p', "Port to run responder on"},
+    {"port", OPT_PORT, 'n', "Port to run responder on"},
```

Comment:
No, `'p'` is correct.  I understand that it may disturb your wishes to test, but "fixing" it this way isn't ok

Reply:
When `ocsp` is used in server mode, "-port 0" can make sense - or why not?

Reply:
Ah!  Change it to `'N'` then.  The problem with `'n'` is that it allows negative numbers.

Reply:
Done